services/core-api/app/api/now_applications/models/applications_view.py

```diff
@@ -0,0 +1,121 @@
+
+    notice_of_work_type_description = db.Column(db.String)
+    now_application_status_description = db.Column(db.String)
+
+    received_date = db.Column(db.Date)
+    originating_system = db.Column(db.String)
+    application_type_code = db.Column(db.String)
+    now_application_status_code = db.Column(db.String)
+    decision_date = db.Column(db.DateTime)
+    source_permit_no = db.Column(db.String)
+
+    is_historic = db.Column(db.Boolean)
+
+    import_timestamp = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    update_timestamp = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+
+    submission_documents = db.relationship(
+        'Document',
+        lazy='selectin',
+        secondary=
+        'join(NOWApplicationIdentity, Document, foreign(NOWApplicationIdentity.messageid)==remote(Document.messageid))',
+        primaryjoin=
+        'and_(ApplicationsView.now_application_guid==NOWApplicationIdentity.now_application_guid, foreign(NOWApplicationIdentity.messageid)==remote(Document.messageid))',
+        secondaryjoin='foreign(NOWApplicationIdentity.messageid)==remote(Document.messageid)',
+        viewonly=True)
+
+    documents = db.relationship(
+        'NOWApplicationDocumentXref',
+        lazy='selectin',
+        primaryjoin=
+        'and_(foreign(NOWApplicationDocumentXref.now_application_id)==ApplicationsView.now_application_id, NOWApplicationDocumentXref.now_application_review_id==None)',
+        order_by='desc(NOWApplicationDocumentXref.create_timestamp)')
+
+    permit_amendments = db.relationship(
+        'PermitAmendment',
+        lazy='select',
+        primaryjoin=
+        'and_(foreign(PermitAmendment.now_application_guid)==ApplicationsView.now_application_guid )'
+    )
+
+    application_reason_codes = db.relationship(
+        'ApplicationReasonCode',
+        lazy='selectin',
+        primaryjoin=
+        'and_(foreign(ApplicationReasonXref.now_application_id)==ApplicationsView.now_application_id)',
+        secondary=
+        'join(ApplicationReasonXref, ApplicationReasonCode, foreign(ApplicationReasonXref.application_reason_code)==remote(ApplicationReasonCode.application_reason_code))',
+        secondaryjoin=
+        'foreign(ApplicationReasonXref.application_reason_code)==remote(ApplicationReasonCode.application_reason_code)',
+        viewonly=True)
+
+    contacts = db.relationship(
+        'NOWPartyAppointment',
+        lazy='selectin',
+        primaryjoin=
+        'and_(foreign(NOWPartyAppointment.now_application_id) == ApplicationsView.now_application_id, NOWPartyAppointment.deleted_ind==False)',
+        secondary=
+        'join(NOWPartyAppointment, Party, foreign(NOWPartyAppointment.party_guid)==remote(Party.party_guid))',
+        secondaryjoin='foreign(NOWPartyAppointment.party_guid)==remote(Party.party_guid)',
+    )
+
+    @hybrid_property
+    def permittee(self):
+        if self.application_type_code == 'NOW':
```

Comment:
NoW's can't have permittees?

Reply:
They can, but we do not need to add this extra info for NOWs, we are not using them and I did not want to extend execution time for search of notice of work